src/rgw/rgw_op.cc

```diff
@@ -6002,6 +6002,17 @@ void RGWDeleteMultiObj::execute()
   if (!multi_delete) {
     op_ret = -EINVAL;
     goto error;
+  } else {
+#define DELETE_MULTI_OBJ_MAX_NUM      1000
+    int max_num = s->cct->_conf->rgw_delete_multi_obj_max_num;
+    if (max_num < 0) {
+      max_num = DELETE_MULTI_OBJ_MAX_NUM;
+    }
+    int multi_delete_object_num = multi_delete->objects.size();
+    if (multi_delete_object_num > max_num) {
+      op_ret = -ERR_MALFORMED_XML;
```

Comment:
@joke-lee MalformedXML doesn't seem like the error AWS would return in this case.  I don't see it documented.  Did you check?

Reply:
hi @mattbenjamin 
my boto3 test script here 
```
from boto3.session import Session
import datetime
import boto3

access_key = ""
secret_key = ""

url = "http://s3.amazonaws.com"

session = Session(access_key, secret_key)
s3_client = session.client('s3', endpoint_url=url)
objects_to_delete = []

for i in range(1, 1002):
  objects_to_delete.append( {'Key':'note_%s.txt' % (i,)} )

#objects_to_delete = [{'Key':'1.txt'}, {'Key':'2.txt'}]
print s3_client.delete_objects(
    Bucket="wzyuliyangbucket01",
    Delete={
        'Objects': objects_to_delete
    }
)

```
and the result

```
Traceback (most recent call last):
  File "deletem.py", line 21, in <module>
    'Objects': objects_to_delete
  File "/usr/lib/python2.7/site-packages/botocore/client.py", line 314, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/usr/lib/python2.7/site-packages/botocore/client.py", line 612, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (MalformedXML) when calling the DeleteObjects operation: The XML you provided was not well-formed or did not validate against our publishedschema

```


here is wireshark

```
POST /wzyuliyangbucket01?delete HTTP/1.1
Host: s3.amazonaws.com
Accept-Encoding: identity
X-Amz-Content-SHA256: dd467bb7ffa5cf5f99994217a5a6293b020a2dd7038d3992142b59ff2f720657
Content-Length: 39999
Content-MD5: rGmlVQwQeBPwu7kiuePUxQ==
X-Amz-Date: 20180813T233349Z
User-Agent: Boto3/1.7.75 Python/2.7.5 Linux/2.6.32-042stab127.2 Botocore/1.10.75
Authorization: AWS4-HMAC-SHA256 Credential=AKIAI62ZIFWLOCODA6RA/20180813/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=72f811c532f1a8c87bf8f351891d512a7af6fd0f9d079f810119025205191072

<Delete xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Object><Key>note_1.txt</Key></Object><Object><Key>note_2.txt</Key></Object><Object><Key>note_3.txt</Key></Object><Object><Key>note_4.txt</Key></Object><Object><Key>note_5.txt</Key></Object><Object><Key>note_6.txt</Key></Object><Object><Key>note_7.txt</Key></Object><Object><Key>note_8.txt</Key></Object><Object><Key>note_9.txt</Key></Object><Object><Key>note_10.txt</Key></Object><Object><Key>note_11.txt</Key></Object><Object><Key>note_12.txt</Key></Object><Object><Key>note_13.txt</Key></Object><Object><Key>note_14.txt</Key></Object><Object><Key>note_15.txt</Key></Object><Object><Key>note_16.txt</Key></Object><Object><Key>note_17.txt</Key></Object><Object><Key>note_18.txt</Key></Object><Object><Key>note_19.txt</Key></Object><Object><Key>note_20.txt</Key></Object><Object><Key>note_21.txt</Key></Object><Object><Key>note_22.txt</Key></Object><Object><Key>note_23.txt</Key></Object><Object><Key>note_24.txt</Key></Object><Object><Key>note_25.txt</Key></Object><Object><Key>note_26.txt</Key></Object><Object><Key>note_27.txt</Key></Object><Object><Key>note_28.txt</Key></Object><Object><Key>note_29.txt</Key></Object><Object><Key>note_30.txt</Key></Object><Object><Key>note_31.txt</Key></Object><Object><Key>note_32.txt</Key></Object><Object><Key>note_33.txt</Key></Object><Object><Key>note_34.txt</Key></Object><Object><Key>note_35.txt</Key></Object><Object><Key>note_36.txt</Key></Object><Object><Key>note_37.txt</Key></Object><Object><Key>note_38.txt</Key></Object><Object><Key>note_39.txt</Key></Object><Object><Key>note_40.txt</Key></Object><Object><Key>note_41.txt</Key></Object><Object><Key>note_42.txt</Key></Object><Object><Key>note_43.txt</Key></Object><Object><Key>note_44.txt</Key></Object><Object><Key>note_45.txt</Key></Object><Object><Key>note_46.txt</Key></Object><Object><Key>note_47.txt</Key></Object><Object><Key>note_48.txt</Key></Object><Object><Key>note_49.txt</Key></Object><Object><Key>note_50.txt</Key></Object><Object><Key>note_51.txt</Key></Object><Object><Key>note_52.txt</Key></Object><Object><Key>note_53.txt</Key></Object><Object><Key>note_54.txt</Key></Object><Object><Key>note_55.txt</Key></Object><Object><Key>note_56.txt</Key></Object><Object><Key>note_57.txt</Key></Object><Object><Key>note_58.txt</Key></Object><Object><Key>note_59.txt</Key></Object><Object><Key>note_60.txt</Key></Object><Object><Key>note_61.txt</Key></Object><Object><Key>note_62.txt</Key></Object><Object><Key>note_63.txt</Key></Object><Object><Key>note_64.txt</Key></Object><Object><Key>note_65.txt</Key></Object><Object><Key>note_66.txt</Key></Object><Object><Key>note_67.txt</Key></Object><Object><Key>note_68.txt</Key></Object><Object><Key>note_69.txt</Key></Object><Object><Key>note_70.txt</Key></Object><Object><Key>note_71.txt</Key></Object><Object><Key>note_72.txt</Key></Object><Object><Key>note_73.txt</Key></Object><Object><Key>note_74.txt</Key></Object><Object><Key>note_75.txt</Key></Object><Object><Key>note_76.txt</Key></Object><Object><Key>note_77.txt</Key></Object><Object><Key>note_78.txt</Key></Object><Object><Key>note_79.txt</Key></Object><Object><Key>note_80.txt</Key></Object><Object><Key>note_81.txt</Key></Object><Object><Key>note_82.txt</Key></Object><Object><Key>note_83.txt</Key></Object><Object><Key>note_84.txt</Key></Object><Object><Key>note_85.txt</Key></Object><Object><Key>note_86.txt</Key></Object><Object><Key>note_87.txt</Key></Object><Object><Key>note_88.txt</Key></Object><Object><Key>note_89.txt</Key></Object><Object><Key>note_90.txt</Key></Object><Object><Key>note_91.txt</Key></Object><Object><Key>note_92.txt</Key></Object><Object><Key>note_93.txt</Key></Object><Object><Key>note_94.txt</Key></Object><Object><Key>note_95.txt</Key></Object><Object><Key>note_96.txt</Key></Object><Object><Key>note_97.txt</Key></Object><Object><Key>note_98.txt</Key></Object><Object><Key>note_99.txt</Key></Object><Object><Key>note_100.txt</Key></Object><Object><Key>note_101.txt</Key></Object><Object><Key>note_102.txt</Key></Object><Object><Key>note_103.txt</Key></Object><Object><Key>note_104.txt</Key></Object><Object><Key>note_105.txt</Key></Object><Object><Key>note_106.txt</Key></Object><Object><Key>note_107.txt</Key></Object><Object><Key>note_108.txt</Key></Object><Object><Key>note_109.txt</Key></Object><Object><Key>note_110.txt</Key></Object><Object><Key>note_111.txt</Key></Object><Object><Key>note_112.txt</Key></Object><Object><Key>note_113.txt</Key></Object><Object><Key>note_114.txt</Key></Object><Object><Key>note_115.txt</Key></Object><Object><Key>note_116.txt</Key></Object><Object><Key>note_117.txt</Key></Object><Object><Key>note_118.txt</Key></Object><Object><Key>note_119.txt</Key></Object><Object><Key>note_120.txt</Key></Object><Object><Key>note_121.txt</Key></Object><Object><Key>note_122.txt</Key></Object><Object><Key>note_123.txt</Key></Object><Object><Key>note_124.txt</Key></Object><Object><Key>note_125.txt</Key></Object><Object><Key>note_126.txt</Key></Object><Object><Key>note_127.txt</Key></Object><Object><Key>note_128.txt</Key></Object><Object><Key>note_129.txt</Key></Object><Object><Key>note_130.txt</Key></Object><Object><Key>note_131.txt</Key></Object><Object><Key>note_132.txt</Key></Object><Object><Key>note_133.txt</Key></Object><Object><Key>note_134.txt</Key></Object><Object><Key>note_135.txt</Key></Object><Object><Key>note_136.txt</Key></Object><Object><Key>note_137.txt</Key></Object><Object><Key>note_138.txt</Key></Object><Object><Key>note_139.txt</Key></Object><Object><Key>note_140.txt</Key></Object><Object><Key>note_141.txt</Key></Object><Object><Key>note_142.txt</Key></Object><Object><Key>note_143.txt</Key></Object><Object><Key>note_144.txt</Key></Object><Object><Key>note_145.txt</Key></Object><Object><Key>note_146.txt</Key></Object><Object><Key>note_147.txt</Key></Object><Object><Key>note_148.txt</Key></Object><Object><Key>note_149.txt</Key></Object><Object><Key>note_150.txt</Key></Object><Object><Key>note_151.txt</Key></Object><Object><Key>note_152.txt</Key></Object><Object><Key>note_153.txt</Key></Object><Object><Key>note_154.txt</Key></Object><Object><Key>note_155.txt</Key></Object><Object><Key>note_156.txt</Key></Object><Object><Key>note_157.txt</Key></Object><Object><Key>note_158.txt</Key></Object><Object><Key>note_159.txt</Key></Object><Object><Key>note_160.txt</Key></Object><Object><Key>note_161.txt</Key></Object><Object><Key>note_162.txt</Key></Object><Object><Key>note_163.txt</Key></Object><Object><Key>note_164.txt</Key></Object><Object><Key>note_165.txt</Key></Object><Object><Key>note_166.txt</Key></Object><Object><Key>note_167.txt</Key></Object><Object><Key>note_168.txt</Key></Object><Object><Key>note_169.txt</Key></Object><Object><Key>note_170.txt</Key></Object><Object><Key>note_171.txt</Key></Object><Object><Key>note_172.txt</Key></Object><Object><Key>note_173.txt</Key></Object><Object><Key>note_174.txt</Key></Object><Object><Key>note_175.txt</Key></Object><Object><Key>note_176.txt</Key></Object><Object><Key>note_177.txt</Key></Object><Object><Key>note_178.txt</Key></Object><Object><Key>note_179.txt</Key></Object><Object><Key>note_180.txt</Key></Object><Object><Key>note_181.txt</Key></Object><Object><Key>note_182.txt</Key></Object><Object><Key>note_183.txt</Key></Object><Object><Key>note_184.txt</Key></Object><Object><Key>note_185.txt</Key></Object><Object><Key>note_186.txt</Key></Object><Object><Key>note_187.txt</Key></Object><Object><Key>note_188.txt</Key></Object><Object><Key>note_189.txt</Key></Object><Object><Key>note_190.txt</Key></Object><Object><Key>note_191.txt</Key></Object><Object><Key>note_192.txt</Key></Object><Object><Key>note_193.txt</Key></Object><Object><Key>note_194.txt</Key></Object><Object><Key>note_195.txt</Key></Object><Object><Key>note_196.txt</Key></Object><Object><Key>note_197.txt</Key></Object><Object><Key>note_198.txt</Key></Object><Object><Key>note_199.txt</Key></Object><Object><Key>note_200.txt</Key></Object><Object><Key>note_201.txt</Key></Object><Object><Key>note_202.txt</Key></Object><Object><Key>note_203.txt</Key></Object><Object><Key>note_204.txt</Key></Object><Object><Key>note_205.txt</Key></Object><Object><Key>note_206.txt</Key></Object><Object><Key>note_207.txt</Key></Object><Object><Key>note_208.txt</Key></Object><Object><Key>note_209.txt</Key></Object><Object><Key>note_210.txt</Key></Object><Object><Key>note_211.txt</Key></Object><Object><Key>note_212.txt</Key></Object><Object><Key>note_213.txt</Key></Object><Object><Key>note_214.txt</Key></Object><Object><Key>note_215.txt</Key></Object><Object><Key>note_216.txt</Key></Object><Object><Key>note_217.txt</Key></Object><Object><Key>note_218.txt</Key></Object><Object><Key>note_219.txt</Key></Object><Object><Key>note_220.txt</Key></Object><Object><Key>note_221.txt</Key></Object><Object><Key>note_222.txt</Key></Object><Object><Key>note_223.txt</Key></Object><Object><Key>note_224.txt</Key></Object><Object><Key>note_225.txt</Key></Object><Object><Key>note_226.txt</Key></Object><Object><Key>note_227.txt</Key></Object><Object><Key>note_228.txt</Key></Object><Object><Key>note_229.txt</Key></Object><Object><Key>note_230.txt</Key></Object><Object><Key>note_231.txt</Key></Object><Object><Key>note_232.txt</Key></Object><Object><Key>note_233.txt</Key></Object><Object><Key>note_234.txt</Key></Object><Object><Key>note_235.txt</Key></Object><Object><Key>note_236.txt</Key></Object><Object><Key>note_237.txt</Key></Object><Object><Key>note_238.txt</Key></Object><Object><Key>note_239.txt</Key></Object><Object><Key>note_240.txt</Key></Object><Object><Key>note_241.txt</Key></Object><Object><Key>note_242.txt</Key></Object><Object><Key>note_243.txt</Key></Object><Object><Key>note_244.txt</Key></Object><Object><Key>note_245.txt</Key></Object><Object><Key>note_246.txt</Key></Object><Object><Key>note_247.txt</Key></Object><Object><Key>note_248.txt</Key></Object><Object><Key>note_249.txt</Key></Object><Object><Key>note_250.txt</Key></Object><Object><Key>note_251.txt</Key></Object><Object><Key>note_252.txt</Key></Object><Object><Key>note_253.txt</Key></Object><Object><Key>note_254.txt</Key></Object><Object><Key>note_255.txt</Key></Object><Object><Key>note_256.txt</Key></Object><Object><Key>note_257.txt</Key></Object><Object><Key>note_258.txt</Key></Object><Object><Key>note_259.txt</Key></Object><Object><Key>note_260.txt</Key></Object><Object><Key>note_261.txt</Key></Object><Object><Key>note_262.txt</Key></Object><Object><Key>note_263.txt</Key></Object><Object><Key>note_264.txt</Key></Object><Object><Key>note_265.txt</Key></Object><Object><Key>note_266.txt</Key></Object><Object><Key>note_267.txt</Key></Object><Object><Key>note_268.txt</Key></Object><Object><Key>note_269.txt</Key></Object><Object><Key>note_270.txt</Key></Object><Object><Key>note_271.txt</Key></Object><Object><Key>note_272.txt</Key></Object><Object><Key>note_273.txt</Key></Object><Object><Key>note_274.txt</Key></Object><Object><Key>note_275.txt</Key></Object><Object><Key>note_276.txt</Key></Object><Object><Key>note_277.txt</Key></Object><Object><Key>note_278.txt</Key></Object><Object><Key>note_279.txt</Key></Object><Object><Key>note_280.txt</Key></Object><Object><Key>note_281.txt</Key></Object><Object><Key>note_282.txt</Key></Object><Object><Key>note_283.txt</Key></Object><Object><Key>note_284.txt</Key></Object><Object><Key>note_285.txt</Key></Object><Object><Key>note_286.txt</Key></Object><Object><Key>note_287.txt</Key></Object><Object><Key>note_288.txt</Key></Object><Object><Key>note_289.txt</Key></Object><Object><Key>note_290.txt</Key></Object><Object><Key>note_291.txt</Key></Object><Object><Key>note_292.txt</Key></Object><Object><Key>note_293.txt</Key></Object><Object><Key>note_294.txt</Key></Object><Object><Key>note_295.txt</Key></Object><Object><Key>note_296.txt</Key></Object><Object><Key>note_297.txt</Key></Object><Object><Key>note_298.txt</Key></Object><Object><Key>note_299.txt</Key></Object><Object><Key>note_300.txt</Key></Object><Object><Key>note_301.txt</Key></Object><Object><Key>note_302.txt</Key></Object><Object><Key>note_303.txt</Key></Object><Object><Key>note_304.txt</Key></Object><Object><Key>note_305.txt</Key></Object><Object><Key>note_306.txt</Key></Object><Object><Key>note_307.txt</Key></Object><Object><Key>note_308.txt</Key></Object><Object><Key>note_309.txt</Key></Object><Object><Key>note_310.txt</Key></Object><Object><Key>note_311.txt</Key></Object><Object><Key>note_312.txt</Key></Object><Object><Key>note_313.txt</Key></Object><Object><Key>note_314.txt</Key></Object><Object><Key>note_315.txt</Key></Object><Object><Key>note_316.txt</Key></Object><Object><Key>note_317.txt</Key></Object><Object><Key>note_318.txt</Key></Object><Object><Key>note_319.txt</Key></Object><Object><Key>note_320.txt</Key></Object><Object><Key>note_321.txt</Key></Object><Object><Key>note_322.txt</Key></Object><Object><Key>note_323.txt</Key></Object><Object><Key>note_324.txt</Key></Object><Object><Key>note_325.txt</Key></Object><Object><Key>note_326.txt</Key></Object><Object><Key>note_327.txt</Key></Object><Object><Key>note_328.txt</Key></Object><Object><Key>note_329.txt</Key></Object><Object><Key>note_330.txt</Key></Object><Object><Key>note_331.txt</Key></Object><Object><Key>note_332.txt</Key></Object><Object><Key>note_333.txt</Key></Object><Object><Key>note_334.txt</Key></Object><Object><Key>note_335.txt</Key></Object><Object><Key>note_336.txt</Key></Object><Object><Key>note_337.txt</Key></Object><Object><Key>note_338.txt</Key></Object><Object><Key>note_339.txt</Key></Object><Object><Key>note_340.txt</Key></Object><Object><Key>note_341.txt</Key></Object><Object><Key>note_342.txt</Key></Object><Object><Key>note_343.txt</Key></Object><Object><Key>note_344.txt</Key></Object><Object><Key>note_345.txt</Key></Object><Object><Key>note_346.txt</Key></Object><Object><Key>note_347.txt</Key></Object><Object><Key>note_348.txt</Key></Object><Object><Key>note_349.txt</Key></Object><Object><Key>note_350.txt</Key></Object><Object><Key>note_351.txt</Key></Object><Object><Key>note_352.txt</Key></Object><Object><Key>note_353.txt</Key></Object><Object><Key>note_354.txt</Key></Object><Object><Key>note_355.txt</Key></Object><Object><Key>note_356.txt</Key></Object><Object><Key>note_357.txt</Key></Object><Object><Key>note_358.txt</Key></Object><Object><Key>note_359.txt</Key></Object><Object><Key>note_360.txt</Key></Object><Object><Key>note_361.txt</Key></Object><Object><Key>note_362.txt</Key></Object><Object><Key>note_363.txt</Key></Object><Object><Key>note_364.txt</Key></Object><Object><Key>note_365.txt</Key></Object><Object><Key>note_366.txt</Key></Object><Object><Key>note_367.txt</Key></Object><Object><Key>note_368.txt</Key></Object><Object><Key>note_369.txt</Key></Object><Object><Key>note_370.txt</Key></Object><Object><Key>note_371.txt</Key></Object><Object><Key>note_372.txt</Key></Object><Object><Key>note_373.txt</Key></Object><Object><Key>note_374.txt</Key></Object><Object><Key>note_375.txt</Key></Object><Object><Key>note_376.txt</Key></Object><Object><Key>note_377.txt</Key></Object><Object><Key>note_378.txt</Key></Object><Object><Key>note_379.txt</Key></Object><Object><Key>note_380.txt</Key></Object><Object><Key>note_381.txt</Key></Object><Object><Key>note_382.txt</Key></Object><Object><Key>note_383.txt</Key></Object><Object><Key>note_384.txt</Key></Object><Object><Key>note_385.txt</Key></Object><Object><Key>note_386.txt</Key></Object><Object><Key>note_387.txt</Key></Object><Object><Key>note_388.txt</Key></Object><Object><Key>note_389.txt</Key></Object><Object><Key>note_390.txt</Key></Object><Object><Key>note_391.txt</Key></Object><Object><Key>note_392.txt</Key></Object><Object><Key>note_393.txt</Key></Object><Object><Key>note_394.txt</Key></Object><Object><Key>note_395.txt</Key></Object><Object><Key>note_396.txt</Key></Object><Object><Key>note_397.txt</Key></Object><Object><Key>note_398.txt</Key></Object><Object><Key>note_399.txt</Key></Object><Object><Key>note_400.txt</Key></Object><Object><Key>note_401.txt</Key></Object><Object><Key>note_402.txt</Key></Object><Object><Key>note_403.txt</Key></Object><Object><Key>note_404.txt</Key></Object><Object><Key>note_405.txt</Key></Object><Object><Key>note_406.txt</Key></Object><Object><Key>note_407.txt</Key></Object><Object><Key>note_408.txt</Key></Object><Object><Key>note_409.txt</Key></Object><Object><Key>note_410.txt</Key></Object><Object><Key>note_411.txt</Key></Object><Object><Key>note_412.txt</Key></Object><Object><Key>note_413.txt</Key></Object><Object><Key>note_414.txt</Key></Object><Object><Key>note_415.txt</Key></Object><Object><Key>note_416.txt</Key></Object><Object><Key>note_417.txt</Key></Object><Object><Key>note_418.txt</Key></Object><Object><Key>note_419.txt</Key></Object><Object><Key>note_420.txt</Key></Object><Object><Key>note_421.txt</Key></Object><Object><Key>note_422.txt</Key></Object><Object><Key>note_423.txt</Key></Object><Object><Key>note_424.txt</Key></Object><Object><Key>note_425.txt</Key></Object><Object><Key>note_426.txt</Key></Object><Object><Key>note_427.txt</Key></Object><Object><Key>note_428.txt</Key></Object><Object><Key>note_429.txt</Key></Object><Object><Key>note_430.txt</Key></Object><Object><Key>note_431.txt</Key></Object><Object><Key>note_432.txt</Key></Object><Object><Key>note_433.txt</Key></Object><Object><Key>note_434.txt</Key></Object><Object><Key>note_435.txt</Key></Object><Object><Key>note_436.txt</Key></Object><Object><Key>note_437.txt</Key></Object><Object><Key>note_438.txt</Key></Object><Object><Key>note_439.txt</Key></Object><Object><Key>note_440.txt</Key></Object><Object><Key>note_441.txt</Key></Object><Object><Key>note_442.txt</Key></Object><Object><Key>note_443.txt</Key></Object><Object><Key>note_444.txt</Key></Object><Object><Key>note_445.txt</Key></Object><Object><Key>note_446.txt</Key></Object><Object><Key>note_447.txt</Key></Object><Object><Key>note_448.txt</Key></Object><Object><Key>note_449.txt</Key></Object><Object><Key>note_450.txt</Key></Object><Object><Key>note_451.txt</Key></Object><Object><Key>note_452.txt</Key></Object><Object><Key>note_453.txt</Key></Object><Object><Key>note_454.txt</Key></Object><Object><Key>note_455.txt</Key></Object><Object><Key>note_456.txt</Key></Object><Object><Key>note_457.txt</Key></Object><Object><Key>note_458.txt</Key></Object><Object><Key>note_459.txt</Key></Object><Object><Key>note_460.txt</Key></Object><Object><Key>note_461.txt</Key></Object><Object><Key>note_462.txt</Key></Object><Object><Key>note_463.txt</Key></Object><Object><Key>note_464.txt</Key></Object><Object><Key>note_465.txt</Key></Object><Object><Key>note_466.txt</Key></Object><Object><Key>note_467.txt</Key></Object><Object><Key>note_468.txt</Key></Object><Object><Key>note_469.txt</Key></Object><Object><Key>note_470.txt</Key></Object><Object><Key>note_471.txt</Key></Object><Object><Key>note_472.txt</Key></Object><Object><Key>note_473.txt</Key></Object><Object><Key>note_474.txt</Key></Object><Object><Key>note_475.txt</Key></Object><Object><Key>note_476.txt</Key></Object><Object><Key>note_477.txt</Key></Object><Object><Key>note_478.txt</Key></Object><Object><Key>note_479.txt</Key></Object><Object><Key>note_480.txt</Key></Object><Object><Key>note_481.txt</Key></Object><Object><Key>note_482.txt</Key></Object><Object><Key>note_483.txt</Key></Object><Object><Key>note_484.txt</Key></Object><Object><Key>note_485.txt</Key></Object><Object><Key>note_486.txt</Key></Object><Object><Key>note_487.txt</Key></Object><Object><Key>note_488.txt</Key></Object><Object><Key>note_489.txt</Key></Object><Object><Key>note_490.txt</Key></Object><Object><Key>note_491.txt</Key></Object><Object><Key>note_492.txt</Key></Object><Object><Key>note_493.txt</Key></Object><Object><Key>note_494.txt</Key></Object><Object><Key>note_495.txt</Key></Object><Object><Key>note_496.txt</Key></Object><Object><Key>note_497.txt</Key></Object><Object><Key>note_498.txt</Key></Object><Object><Key>note_499.txt</Key></Object><Object><Key>note_500.txt</Key></Object><Object><Key>note_501.txt</Key></Object><Object><Key>note_502.txt</Key></Object><Object><Key>note_503.txt</Key></Object><Object><Key>note_504.txt</Key></Object><Object><Key>note_505.txt</Key></Object><Object><Key>note_506.txt</Key></Object><Object><Key>note_507.txt</Key></Object><Object><Key>note_508.txt</Key></Object><Object><Key>note_509.txt</Key></Object><Object><Key>note_510.txt</Key></Object><Object><Key>note_511.txt</Key></Object><Object><Key>note_512.txt</Key></Object><Object><Key>note_513.txt</Key></Object><Object><Key>note_514.txt</Key></Object><Object><Key>note_515.txt</Key></Object><Object><Key>note_516.txt</Key></Object><Object><Key>note_517.txt</Key></Object><Object><Key>note_518.txt</Key></Object><Object><Key>note_519.txt</Key></Object><Object><Key>note_520.txt</Key></Object><Object><Key>note_521.txt</Key></Object><Object><Key>note_522.txt</Key></Object><Object><Key>note_523.txt</Key></Object><Object><Key>note_524.txt</Key></Object><Object><Key>note_525.txt</Key></Object><Object><Key>note_526.txt</Key></Object><Object><Key>note_527.txt</Key></Object><Object><Key>note_528.txt</Key></Object><Object><Key>note_529.txt</Key></Object><Object><Key>note_530.txt</Key></Object><Object><Key>note_531.txt</Key></Object><Object><Key>note_532.txt</Key></Object><Object><Key>note_533.txt</Key></Object><Object><Key>note_534.txt</Key></Object><Object><Key>note_535.txt</Key></Object><Object><Key>note_536.txt</Key></Object><Object><Key>note_537.txt</Key></Object><Object><Key>note_538.txt</Key></Object><Object><Key>note_539.txt</Key></Object><Object><Key>note_540.txt</Key></Object><Object><Key>note_541.txt</Key></Object><Object><Key>note_542.txt</Key></Object><Object><Key>note_543.txt</Key></Object><Object><Key>note_544.txt</Key></Object><Object><Key>note_545.txt</Key></Object><Object><Key>note_546.txt</Key></Object><Object><Key>note_547.txt</Key></Object><Object><Key>note_548.txt</Key></Object><Object><Key>note_549.txt</Key></Object><Object><Key>note_550.txt</Key></Object><Object><Key>note_551.txt</Key></Object><Object><Key>note_552.txt</Key></Object><Object><Key>note_553.txt</Key></Object><Object><Key>note_554.txt</Key></Object><Object><Key>note_555.txt</Key></Object><Object><Key>note_556.txt</Key></Object><Object><Key>note_557.txt</Key></Object><Object><Key>note_558.txt</Key></Object><Object><Key>note_559.txt</Key></Object><Object><Key>note_560.txt</Key></Object><Object><Key>note_561.txt</Key></Object><Object><Key>note_562.txt</Key></Object><Object><Key>note_563.txt</Key></Object><Object><Key>note_564.txt</Key></Object><Object><Key>note_565.txt</Key></Object><Object><Key>note_566.txt</Key></Object><Object><Key>note_567.txt</Key></Object><Object><Key>note_568.txt</Key></Object><Object><Key>note_569.txt</Key></Object><Object><Key>note_570.txt</Key></Object><Object><Key>note_571.txt</Key></Object><Object><Key>note_572.txt</Key></Object><Object><Key>note_573.txt</Key></Object><Object><Key>note_574.txt</Key></Object><Object><Key>note_575.txt</Key></Object><Object><Key>note_576.txt</Key></Object><Object><Key>note_577.txt</Key></Object><Object><Key>note_578.txt</Key></Object><Object><Key>note_579.txt</Key></Object><Object><Key>note_580.txt</Key></Object><Object><Key>note_581.txt</Key></Object><Object><Key>note_582.txt</Key></Object><Object><Key>note_583.txt</Key></Object><Object><Key>note_584.txt</Key></Object><Object><Key>note_585.txt</Key></Object><Object><Key>note_586.txt</Key></Object><Object><Key>note_587.txt</Key></Object><Object><Key>note_588.txt</Key></Object><Object><Key>note_589.txt</Key></Object><Object><Key>note_590.txt</Key></Object><Object><Key>note_591.txt</Key></Object><Object><Key>note_592.txt</Key></Object><Object><Key>note_593.txt</Key></Object><Object><Key>note_594.txt</Key></Object><Object><Key>note_595.txt</Key></Object><Object><Key>note_596.txt</Key></Object><Object><Key>note_597.txt</Key></Object><Object><Key>note_598.txt</Key></Object><Object><Key>note_599.txt</Key></Object><Object><Key>note_600.txt</Key></Object><Object><Key>note_601.txt</Key></Object><Object><Key>note_602.txt</Key></Object><Object><Key>note_603.txt</Key></Object><Object><Key>note_604.txt</Key></Object><Object><Key>note_605.txt</Key></Object><Object><Key>note_606.txt</Key></Object><Object><Key>note_607.txt</Key></Object><Object><Key>note_608.txt</Key></Object><Object><Key>note_609.txt</Key></Object><Object><Key>note_610.txt</Key></Object><Object><Key>note_611.txt</Key></Object><Object><Key>note_612.txt</Key></Object><Object><Key>note_613.txt</Key></Object><Object><Key>note_614.txt</Key></Object><Object><Key>note_615.txt</Key></Object><Object><Key>note_616.txt</Key></Object><Object><Key>note_617.txt</Key></Object><Object><Key>note_618.txt</Key></Object><Object><Key>note_619.txt</Key></Object><Object><Key>note_620.txt</Key></Object><Object><Key>note_621.txt</Key></Object><Object><Key>note_622.txt</Key></Object><Object><Key>note_623.txt</Key></Object><Object><Key>note_624.txt</Key></Object><Object><Key>note_625.txt</Key></Object><Object><Key>note_626.txt</Key></Object><Object><Key>note_627.txt</Key></Object><Object><Key>note_628.txt</Key></Object><Object><Key>note_629.txt</Key></Object><Object><Key>note_630.txt</Key></Object><Object><Key>note_631.txt</Key></Object><Object><Key>note_632.txt</Key></Object><Object><Key>note_633.txt</Key></Object><Object><Key>note_634.txt</Key></Object><Object><Key>note_635.txt</Key></Object><Object><Key>note_636.txt</Key></Object><Object><Key>note_637.txt</Key></Object><Object><Key>note_638.txt</Key></Object><Object><Key>note_639.txt</Key></Object><Object><Key>note_640.txt</Key></Object><Object><Key>note_641.txt</Key></Object><Object><Key>note_642.txt</Key></Object><Object><Key>note_643.txt</Key></Object><Object><Key>note_644.txt</Key></Object><Object><Key>note_645.txt</Key></Object><Object><Key>note_646.txt</Key></Object><Object><Key>note_647.txt</Key></Object><Object><Key>note_648.txt</Key></Object><Object><Key>note_649.txt</Key></Object><Object><Key>note_650.txt</Key></Object><Object><Key>note_651.txt</Key></Object><Object><Key>note_652.txt</Key></Object><Object><Key>note_653.txt</Key></Object><Object><Key>note_654.txt</Key></Object><Object><Key>note_655.txt</Key></Object><Object><Key>note_656.txt</Key></Object><Object><Key>note_657.txt</Key></Object><Object><Key>note_658.txt</Key></Object><Object><Key>note_659.txt</Key></Object><Object><Key>note_660.txt</Key></Object><Object><Key>note_661.txt</Key></Object><Object><Key>note_662.txt</Key></Object><Object><Key>note_663.txt</Key></Object><Object><Key>note_664.txt</Key></Object><Object><Key>note_665.txt</Key></Object><Object><Key>note_666.txt</Key></Object><Object><Key>note_667.txt</Key></Object><Object><Key>note_668.txt</Key></Object><Object><Key>note_669.txt</Key></Object><Object><Key>note_670.txt</Key></Object><Object><Key>note_671.txt</Key></Object><Object><Key>note_672.txt</Key></Object><Object><Key>note_673.txt</Key></Object><Object><Key>note_674.txt</Key></Object><Object><Key>note_675.txt</Key></Object><Object><Key>note_676.txt</Key></Object><Object><Key>note_677.txt</Key></Object><Object><Key>note_678.txt</Key></Object><Object><Key>note_679.txt</Key></Object><Object><Key>note_680.txt</Key></Object><Object><Key>note_681.txt</Key></Object><Object><Key>note_682.txt</Key></Object><Object><Key>note_683.txt</Key></Object><Object><Key>note_684.txt</Key></Object><Object><Key>note_685.txt</Key></Object><Object><Key>note_686.txt</Key></Object><Object><Key>note_687.txt</Key></Object><Object><Key>note_688.txt</Key></Object><Object><Key>note_689.txt</Key></Object><Object><Key>note_690.txt</Key></Object><Object><Key>note_691.txt</Key></Object><Object><Key>note_692.txt</Key></Object><Object><Key>note_693.txt</Key></Object><Object><Key>note_694.txt</Key></Object><Object><Key>note_695.txt</Key></Object><Object><Key>note_696.txt</Key></Object><Object><Key>note_697.txt</Key></Object><Object><Key>note_698.txt</Key></Object><Object><Key>note_699.txt</Key></Object><Object><Key>note_700.txt</Key></Object><Object><Key>note_701.txt</Key></Object><Object><Key>note_702.txt</Key></Object><Object><Key>note_703.txt</Key></Object><Object><Key>note_704.txt</Key></Object><Object><Key>note_705.txt</Key></Object><Object><Key>note_706.txt</Key></Object><Object><Key>note_707.txt</Key></Object><Object><Key>note_708.txt</Key></Object><Object><Key>note_709.txt</Key></Object><Object><Key>note_710.txt</Key></Object><Object><Key>note_711.txt</Key></Object><Object><Key>note_712.txt</Key></Object><Object><Key>note_713.txt</Key></Object><Object><Key>note_714.txt</Key></Object><Object><Key>note_715.txt</Key></Object><Object><Key>note_716.txt</Key></Object><Object><Key>note_717.txt</Key></Object><Object><Key>note_718.txt</Key></Object><Object><Key>note_719.txt</Key></Object><Object><Key>note_720.txt</Key></Object><Object><Key>note_721.txt</Key></Object><Object><Key>note_722.txt</Key></Object><Object><Key>note_723.txt</Key></Object><Object><Key>note_724.txt</Key></Object><Object><Key>note_725.txt</Key></Object><Object><Key>note_726.txt</Key></Object><Object><Key>note_727.txt</Key></Object><Object><Key>note_728.txt</Key></Object><Object><Key>note_729.txt</Key></Object><Object><Key>note_730.txt</Key></Object><Object><Key>note_731.txt</Key></Object><Object><Key>note_732.txt</Key></Object><Object><Key>note_733.txt</Key></Object><Object><Key>note_734.txt</Key></Object><Object><Key>note_735.txt</Key></Object><Object><Key>note_736.txt</Key></Object><Object><Key>note_737.txt</Key></Object><Object><Key>note_738.txt</Key></Object><Object><Key>note_739.txt</Key></Object><Object><Key>note_740.txt</Key></Object><Object><Key>note_741.txt</Key></Object><Object><Key>note_742.txt</Key></Object><Object><Key>note_743.txt</Key></Object><Object><Key>note_744.txt</Key></Object><Object><Key>note_745.txt</Key></Object><Object><Key>note_746.txt</Key></Object><Object><Key>note_747.txt</Key></Object><Object><Key>note_748.txt</Key></Object><Object><Key>note_749.txt</Key></Object><Object><Key>note_750.txt</Key></Object><Object><Key>note_751.txt</Key></Object><Object><Key>note_752.txt</Key></Object><Object><Key>note_753.txt</Key></Object><Object><Key>note_754.txt</Key></Object><Object><Key>note_755.txt</Key></Object><Object><Key>note_756.txt</Key></Object><Object><Key>note_757.txt</Key></Object><Object><Key>note_758.txt</Key></Object><Object><Key>note_759.txt</Key></Object><Object><Key>note_760.txt</Key></Object><Object><Key>note_761.txt</Key></Object><Object><Key>note_762.txt</Key></Object><Object><Key>note_763.txt</Key></Object><Object><Key>note_764.txt</Key></Object><Object><Key>note_765.txt</Key></Object><Object><Key>note_766.txt</Key></Object><Object><Key>note_767.txt</Key></Object><Object><Key>note_768.txt</Key></Object><Object><Key>note_769.txt</Key></Object><Object><Key>note_770.txt</Key></Object><Object><Key>note_771.txt</Key></Object><Object><Key>note_772.txt</Key></Object><Object><Key>note_773.txt</Key></Object><Object><Key>note_774.txt</Key></Object><Object><Key>note_775.txt</Key></Object><Object><Key>note_776.txt</Key></Object><Object><Key>note_777.txt</Key></Object><Object><Key>note_778.txt</Key></Object><Object><Key>note_779.txt</Key></Object><Object><Key>note_780.txt</Key></Object><Object><Key>note_781.txt</Key></Object><Object><Key>note_782.txt</Key></Object><Object><Key>note_783.txt</Key></Object><Object><Key>note_784.txt</Key></Object><Object><Key>note_785.txt</Key></Object><Object><Key>note_786.txt</Key></Object><Object><Key>note_787.txt</Key></Object><Object><Key>note_788.txt</Key></Object><Object><Key>note_789.txt</Key></Object><Object><Key>note_790.txt</Key></Object><Object><Key>note_791.txt</Key></Object><Object><Key>note_792.txt</Key></Object><Object><Key>note_793.txt</Key></Object><Object><Key>note_794.txt</Key></Object><Object><Key>note_795.txt</Key></Object><Object><Key>note_796.txt</Key></Object><Object><Key>note_797.txt</Key></Object><Object><Key>note_798.txt</Key></Object><Object><Key>note_799.txt</Key></Object><Object><Key>note_800.txt</Key></Object><Object><Key>note_801.txt</Key></Object><Object><Key>note_802.txt</Key></Object><Object><Key>note_803.txt</Key></Object><Object><Key>note_804.txt</Key></Object><Object><Key>note_805.txt</Key></Object><Object><Key>note_806.txt</Key></Object><Object><Key>note_807.txt</Key></Object><Object><Key>note_808.txt</Key></Object><Object><Key>note_809.txt</Key></Object><Object><Key>note_810.txt</Key></Object><Object><Key>note_811.txt</Key></Object><Object><Key>note_812.txt</Key></Object><Object><Key>note_813.txt</Key></Object><Object><Key>note_814.txt</Key></Object><Object><Key>note_815.txt</Key></Object><Object><Key>note_816.txt</Key></Object><Object><Key>note_817.txt</Key></Object><Object><Key>note_818.txt</Key></Object><Object><Key>note_819.txt</Key></Object><Object><Key>note_820.txt</Key></Object><Object><Key>note_821.txt</Key></Object><Object><Key>note_822.txt</Key></Object><Object><Key>note_823.txt</Key></Object><Object><Key>note_824.txt</Key></Object><Object><Key>note_825.txt</Key></Object><Object><Key>note_826.txt</Key></Object><Object><Key>note_827.txt</Key></Object><Object><Key>note_828.txt</Key></Object><Object><Key>note_829.txt</Key></Object><Object><Key>note_830.txt</Key></Object><Object><Key>note_831.txt</Key></Object><Object><Key>note_832.txt</Key></Object><Object><Key>note_833.txt</Key></Object><Object><Key>note_834.txt</Key></Object><Object><Key>note_835.txt</Key></Object><Object><Key>note_836.txt</Key></Object><Object><Key>note_837.txt</Key></Object><Object><Key>note_838.txt</Key></Object><Object><Key>note_839.txt</Key></Object><Object><Key>note_840.txt</Key></Object><Object><Key>note_841.txt</Key></Object><Object><Key>note_842.txt</Key></Object><Object><Key>note_843.txt</Key></Object><Object><Key>note_844.txt</Key></Object><Object><Key>note_845.txt</Key></Object><Object><Key>note_846.txt</Key></Object><Object><Key>note_847.txt</Key></Object><Object><Key>note_848.txt</Key></Object><Object><Key>note_849.txt</Key></Object><Object><Key>note_850.txt</Key></Object><Object><Key>note_851.txt</Key></Object><Object><Key>note_852.txt</Key></Object><Object><Key>note_853.txt</Key></Object><Object><Key>note_854.txt</Key></Object><Object><Key>note_855.txt</Key></Object><Object><Key>note_856.txt</Key></Object><Object><Key>note_857.txt</Key></Object><Object><Key>note_858.txt</Key></Object><Object><Key>note_859.txt</Key></Object><Object><Key>note_860.txt</Key></Object><Object><Key>note_861.txt</Key></Object><Object><Key>note_862.txt</Key></Object><Object><Key>note_863.txt</Key></Object><Object><Key>note_864.txt</Key></Object><Object><Key>note_865.txt</Key></Object><Object><Key>note_866.txt</Key></Object><Object><Key>note_867.txt</Key></Object><Object><Key>note_868.txt</Key></Object><Object><Key>note_869.txt</Key></Object><Object><Key>note_870.txt</Key></Object><Object><Key>note_871.txt</Key></Object><Object><Key>note_872.txt</Key></Object><Object><Key>note_873.txt</Key></Object><Object><Key>note_874.txt</Key></Object><Object><Key>note_875.txt</Key></Object><Object><Key>note_876.txt</Key></Object><Object><Key>note_877.txt</Key></Object><Object><Key>note_878.txt</Key></Object><Object><Key>note_879.txt</Key></Object><Object><Key>note_880.txt</Key></Object><Object><Key>note_881.txt</Key></Object><Object><Key>note_882.txt</Key></Object><Object><Key>note_883.txt</Key></Object><Object><Key>note_884.txt</Key></Object><Object><Key>note_885.txt</Key></Object><Object><Key>note_886.txt</Key></Object><Object><Key>note_887.txt</Key></Object><Object><Key>note_888.txt</Key></Object><Object><Key>note_889.txt</Key></Object><Object><Key>note_890.txt</Key></Object><Object><Key>note_891.txt</Key></Object><Object><Key>note_892.txt</Key></Object><Object><Key>note_893.txt</Key></Object><Object><Key>note_894.txt</Key></Object><Object><Key>note_895.txt</Key></Object><Object><Key>note_896.txt</Key></Object><Object><Key>note_897.txt</Key></Object><Object><Key>note_898.txt</Key></Object><Object><Key>note_899.txt</Key></Object><Object><Key>note_900.txt</Key></Object><Object><Key>note_901.txt</Key></Object><Object><Key>note_902.txt</Key></Object><Object><Key>note_903.txt</Key></Object><Object><Key>note_904.txt</Key></Object><Object><Key>note_905.txt</Key></Object><Object><Key>note_906.txt</Key></Object><Object><Key>note_907.txt</Key></Object><Object><Key>note_908.txt</Key></Object><Object><Key>note_909.txt</Key></Object><Object><Key>note_910.txt</Key></Object><Object><Key>note_911.txt</Key></Object><Object><Key>note_912.txt</Key></Object><Object><Key>note_913.txt</Key></Object><Object><Key>note_914.txt</Key></Object><Object><Key>note_915.txt</Key></Object><Object><Key>note_916.txt</Key></Object><Object><Key>note_917.txt</Key></Object><Object><Key>note_918.txt</Key></Object><Object><Key>note_919.txt</Key></Object><Object><Key>note_920.txt</Key></Object><Object><Key>note_921.txt</Key></Object><Object><Key>note_922.txt</Key></Object><Object><Key>note_923.txt</Key></Object><Object><Key>note_924.txt</Key></Object><Object><Key>note_925.txt</Key></Object><Object><Key>note_926.txt</Key></Object><Object><Key>note_927.txt</Key></Object><Object><Key>note_928.txt</Key></Object><Object><Key>note_929.txt</Key></Object><Object><Key>note_930.txt</Key></Object><Object><Key>note_931.txt</Key></Object><Object><Key>note_932.txt</Key></Object><Object><Key>note_933.txt</Key></Object><Object><Key>note_934.txt</Key></Object><Object><Key>note_935.txt</Key></Object><Object><Key>note_936.txt</Key></Object><Object><Key>note_937.txt</Key></Object><Object><Key>note_938.txt</Key></Object><Object><Key>note_939.txt</Key></Object><Object><Key>note_940.txt</Key></Object><Object><Key>note_941.txt</Key></Object><Object><Key>note_942.txt</Key></Object><Object><Key>note_943.txt</Key></Object><Object><Key>note_944.txt</Key></Object><Object><Key>note_945.txt</Key></Object><Object><Key>note_946.txt</Key></Object><Object><Key>note_947.txt</Key></Object><Object><Key>note_948.txt</Key></Object><Object><Key>note_949.txt</Key></Object><Object><Key>note_950.txt</Key></Object><Object><Key>note_951.txt</Key></Object><Object><Key>note_952.txt</Key></Object><Object><Key>note_953.txt</Key></Object><Object><Key>note_954.txt</Key></Object><Object><Key>note_955.txt</Key></Object><Object><Key>note_956.txt</Key></Object><Object><Key>note_957.txt</Key></Object><Object><Key>note_958.txt</Key></Object><Object><Key>note_959.txt</Key></Object><Object><Key>note_960.txt</Key></Object><Object><Key>note_961.txt</Key></Object><Object><Key>note_962.txt</Key></Object><Object><Key>note_963.txt</Key></Object><Object><Key>note_964.txt</Key></Object><Object><Key>note_965.txt</Key></Object><Object><Key>note_966.txt</Key></Object><Object><Key>note_967.txt</Key></Object><Object><Key>note_968.txt</Key></Object><Object><Key>note_969.txt</Key></Object><Object><Key>note_970.txt</Key></Object><Object><Key>note_971.txt</Key></Object><Object><Key>note_972.txt</Key></Object><Object><Key>note_973.txt</Key></Object><Object><Key>note_974.txt</Key></Object><Object><Key>note_975.txt</Key></Object><Object><Key>note_976.txt</Key></Object><Object><Key>note_977.txt</Key></Object><Object><Key>note_978.txt</Key></Object><Object><Key>note_979.txt</Key></Object><Object><Key>note_980.txt</Key></Object><Object><Key>note_981.txt</Key></Object><Object><Key>note_982.txt</Key></Object><Object><Key>note_983.txt</Key></Object><Object><Key>note_984.txt</Key></Object><Object><Key>note_985.txt</Key></Object><Object><Key>note_986.txt</Key></Object><Object><Key>note_987.txt</Key></Object><Object><Key>note_988.txt</Key></Object><Object><Key>note_989.txt</Key></Object><Object><Key>note_990.txt</Key></Object><Object><Key>note_991.txt</Key></Object><Object><Key>note_992.txt</Key></Object><Object><Key>note_993.txt</Key></Object><Object><Key>note_994.txt</Key></Object><Object><Key>note_995.txt</Key></Object><Object><Key>note_996.txt</Key></Object><Object><Key>note_997.txt</Key></Object><Object><Key>note_998.txt</Key></Object><Object><Key>note_999.txt</Key></Object><Object><Key>note_1000.txt</Key></Object><Object><Key>note_1001.txt</Key></Object></Delete>HTTP/1.1 400 Bad Request
x-amz-request-id: 7C5C432C8397106C
x-amz-id-2: ydyQ54ZL2MjUsPkcTpy8PdJwRRYuAwvb38PQEd11s9nhCuKWiO/nKnOBk84LsqXy4oTFvX4KV6U=
Content-Type: application/xml
Transfer-Encoding: chunked
Date: Mon, 13 Aug 2018 23:33:49 GMT
Connection: close
Server: AmazonS3

<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>MalformedXML</Code><Message>The XML you provided was not well-formed or did not validate against our published schema</Message><RequestId>7C5C432C8397106C</RequestId><HostId>ydyQ54ZL2MjUsPkcTpy8PdJwRRYuAwvb38PQEd11s9nhCuKWiO/nKnOBk84LsqXy4oTFvX4KV6U=</HostId></Error>
```